bin/github-review.sh

```diff
@@ -45,12 +45,40 @@ if [[ -z $chainid ]]; then
     echo "ERROR: Chain ID not specified as per the PR Template" 1>&2
     exit 1
 fi
-chainInfo="https://raw.githubusercontent.com/ethereum-lists/chains/refs/heads/master/_data/chains/eip155-$chainid.json"
-rpc="$(curl -sfL "$chainInfo" | jq -r '.rpc[0]')"
-if [[ -z $rpc ]]; then
-    echo "ERROR: RPC not fetched correctly from the ethereum-lists" 1>&2
+
+# Fetch RPC from DefiLlama's chainlist
+chainlist_response=$(curl -sfL 'https://chainlist.org/rpcs.json' || echo "CURL_ERROR:$?")
+if [[ $chainlist_response == CURL_ERROR:* ]]; then
+    error_code=${chainlist_response#CURL_ERROR:}
+    echo "ERROR: Failed to fetch chainlist from DefiLlama (curl error $error_code)" 1>&2
+    exit 1
+fi
```

Comment:
nit: you can use the same pattern as below, the exit code from `-f` is not very informative


```suggestion
if ! chainlist_response="$(curl -sfL 'https://chainlist.org/rpcs.json')"; then
    echo "ERROR: Failed to fetch DeFiLlama ChainList" 1>&2
    exit 1
fi
```